test/test_torch.py

```diff
@@ -9699,9 +9694,13 @@ def test_rpow(self, device):
     @skipCPUIfNoLapack
     @dtypesIfCPU(torch.float32, torch.float64, torch.complex64, torch.complex128)
     @dtypesIfCUDA(torch.float32, torch.float64)
+    @tf32_on_and_off()
     def test_symeig(self, device, dtype):
         from torch.testing._internal.common_utils import random_hermitian_matrix
 
+        is_tf32 = (dtype == torch.float32 and tf32_is_not_fp32() and torch.backends.cuda.matmul.allow_tf32)
+        recon_tol = 5e-3 if is_tf32 else 1e-8
```

Comment:
What does "recon_tol" mean?

Reply:
I renamed it to `reconstruct_tol`, which is the tolerance for reconstructing from eigen decomposition